theano/tensor/nnet/nnet.py

```diff
@@ -29,6 +29,7 @@
 from theano.tensor.nnet.sigm import sigmoid, softplus
 from theano.gradient import DisconnectedType
 from theano.gradient import grad_not_implemented
+from theano.sandbox.blocksparse import sparse_block_dot
```

Comment:
In theory, everything not well finished/tested/reviewed.

In practice, there is some stuff we should move out of sandbox... I think this part of the code is well tested/reviewed now, so it can be moved out of sandbox.